apps/dashboard/app/javascript/packs/application.js

```diff
@@ -14,32 +14,34 @@
 // const images = require.context('../images', true)
 // const imagePath = (name) => images(name, true)
 
-import 'jquery';
-import 'jquery-ujs'
-
-// lot's of inline scripts and stuff rely on jquery just being available
-window.jQuery = jQuery;
-window.$ = $;
+import jQuery from 'jquery';
+import 'jquery-ujs';
+import datatables from 'datatables.net';
+import datatablesBs4 from 'datatables.net-bs4/js/dataTables.bootstrap4';
+import datatablesSelect from 'datatables.net-select';
 
 import Rails from '@rails/ujs';
-Rails.start();
 
 // Import popper.js for Bootstrap 4
-import 'popper.js'
+import 'popper.js';
 
 // Import Bootstrap 4
-import 'bootstrap/dist/js/bootstrap'
+import 'bootstrap/dist/js/bootstrap';
+
+// FIXME: confim modals don't work in esbuild.
+// import 'data-confirm-modal';
 
-// confim modals
-import 'data-confirm-modal';
+import { setNavbarColor } from './config';
 
-// Import application stylesheets
-import '../stylesheets/application'
+// lot's of inline scripts and stuff rely on jquery just being available
+window.jQuery = jQuery;
+window.$ = jQuery;
 
-import '../images/OpenOnDemand_powered_by_RGB.svg';
-import '../images/OpenOnDemand_stack_RGB.svg';
+datatables(window, jQuery);
```

Comment:
This deserves a comment which I'll give it, but this is here to initialize these plugins. They don't directly initialize just by being imported (because of javascript toolchain compilations and how old it is and how it's being invoked).

So I had to initialize them to tie them to this jquery otherwise they never invoke.